### PR TITLE
SRFCMSAL-345/346/379 new play button for videos

### PR DESCRIPTION
**Pull request by mms-pabo**

### CODE REVIEW

**Comment by claudio-walser** [comment]
retest this please

File: source/_patterns/20-molecules/media-caption/media-caption.scss [hunk 1]
```diff
@@ -158,3 +119,65 @@
...
+  .media-caption__source,
+  .media-caption__source-title {
+    color: $color-border;
+  }
+
+  .media-caption__title,
+  .media-caption__source,
+  .media-caption__source-title {
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
```
Comment (alebon) [inline source/_patterns/20-molecules/media-caption/media-caption.scss]: Mixin, DRY

File: source/_patterns/20-molecules/medium/medium.scss [hunk 2]
```diff
@@ -32,7 +34,13 @@
 
 .medium__caption {
   flex-grow: 1;
-  display: flex;
+  flex-basis: 0%;
+  flex-shrink: 1;
```
Comment (alebon) [inline source/_patterns/20-molecules/medium/medium.scss]: Combine in 1 line

File: source/_patterns/_shame.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+/*
+ * TARGETED BROWSER: IE 11 and Edge <= 15
+ *
+ * This styles are needed for image--media-still classes on which the object-fit-fallback of srf-shame.js is applied.
+ * Parents of elements with those classes usually restrict the aspect ratio of its box. Without the following rules
+ * the hight of the fallback-<div> would be 0.
+ *
+ */
+.image--media-still,
+.image--media-still-video {
```
Comment (alebon) [inline source/_patterns/_shame.scss]: This (maybe) breaks listicle/listing? (-Phil)

File: source/_patterns/20-molecules/media-caption/media-caption.scss (hunk 1 above)
Comment (mms-pabo) [reply source/_patterns/20-molecules/media-caption/media-caption.scss]: done (in a new branch)

File: source/_patterns/20-molecules/medium/medium.scss (hunk 2 above)
Comment (mms-pabo) [reply source/_patterns/20-molecules/medium/medium.scss]: done (in a new branch)

File: source/_patterns/_shame.scss (hunk 3 above)
Comment (mms-pabo) [reply source/_patterns/_shame.scss]: it's only a part of the problem - solved in a new branch

**Pull request merged**